applications/matter_bridge/README.rst

```diff
@@ -0,0 +1,315 @@
+
+By default, the Matter accessory device has no IPv6 network configured.
+To use the device within a Wi-Fi network, you must pair it with the Matter controller over Bluetooth® LE to get the configuration from the controller.
+
+The Bluetooth LE advertising starts automatically upon device startup, but only for a predefined period of time (15 minutes by default).
+If the Bluetooth LE advertising times out, you can re-enable it manually by pressing **Button (SW2)**.
+
+Additionally, the controller must get the `Onboarding information`_ from the Matter accessory device and provision the device into the network.
+For details, see the `Testing`_ section.
+
+.. _matter_bridge_app_build_types:
+
+Matter bridge build types
+=========================
+
+The bridge application uses different configuration files depending on the supported features.
+Configuration files are provided for different build types and they are located in the application root directory.
+
+The :file:`prj.conf` file represents a ``debug`` build type.
+Other build types are covered by dedicated files with the build type added as a suffix to the ``prj`` part, as per the following list.
+For example, the ``release`` build type file name is :file:`prj_release.conf`.
+If a board has other configuration files, for example associated with partition layout or child image configuration, these follow the same pattern.
+
+.. include:: /config_and_build/modifying.rst
+   :start-after: build_types_overview_start
+   :end-before: build_types_overview_end
+
+Before you start testing the application, you can select one of the build types supported by the sample.
+This sample supports the following build types:
+
+* ``debug`` -- Debug version of the application.
+  You can use this version to enable additional features for verifying the application behavior, such as logs.
+* ``release`` -- Release version of the application.
+  You can use this version to enable only the necessary application functionalities to optimize its performance.
+
+.. note::
+    `Selecting a build type`_ is optional.
+    The ``debug`` build type is used by default if no build type is explicitly selected.
+
+User interface
+**************
+
+.. include:: ../../samples/matter/lock/README.rst
+    :start-after: matter_door_lock_sample_led1_start
+    :end-before: matter_door_lock_sample_led1_end
+
+Button 1:
+     If pressed for six seconds, it initiates the factory reset of the device.
+     Releasing the button within the six-second window cancels the factory reset procedure.
+
+Button 2:
+     Enables Bluetooth LE advertising for the predefined period of time (15 minutes by default), and makes the device discoverable over Bluetooth LE.
+
+.. include:: ../lock/README.rst
+    :start-after: matter_door_lock_sample_jlink_start
+    :end-before: matter_door_lock_sample_jlink_end
+
+.. _matter_bridge_cli:
+
+Matter CLI commands
+===================
+
+You can use the following commands to control the bridge device:
+
```

Comment:
In the doc output, it will be hard to distinguish information about the two commands. I suggest you add lines 108-130 in a bullet point, and lines 132-146 in another. You can have:
```suggestion
You can use the following commands to control the bridge device:

* For adding a new bridged device to the bridge:

...

* For removes a new bridged device from the bridge:

```

Reply:
Ok, I think it was like this before the review from @wiba-nordic, so can we have some agreement on that?

Reply:
Or maybe not, it wasn't the same. Ok I will introduce it :)

Reply:
Ok, done, I hope it is correct now.